dartagnan/src/main/java/com/dat3m/dartagnan/program/arch/linux/event/cond/RMWReadCond.java

```diff
@@ -32,8 +29,8 @@
     }
```

Comment:
shouldn't this class override `cfImpliesExec()`?

Reply:
You fell into the same trap as I did. Those events are not conditional, their condition is only checked by the associated `RMWStoreCond`. The design sucks.